schema/schema.go

```diff
@@ -0,0 +1,70 @@
+package schema
+
+import (
+	"os"
+	"strings"
+
+	"github.com/sketch-hq/gql-lint/introspection"
+	"github.com/sketch-hq/gql-lint/parser"
+	"github.com/vektah/gqlparser/v2/ast"
+	gqlvalidator "github.com/vektah/gqlparser/v2/validator"
+)
+
+const deprecated = `directive @deprecated(
+	reason: String = "No longer supported"
+) on FIELD_DEFINITION | ARGUMENT_DEFINITION | ENUM_VALUE | INPUT_FIELD_DEFINITION`
+
+func Load(source string) (*ast.Schema, error) {
+	loader := schemaSources(source)
+	sources, err := loader.Load(source)
+	if err != nil {
+		return nil, err
+	}
+
+	return parser.ParseSchema(sources...)
+}
+
+type SchemaSource interface {
```

Comment:
wdyt if we rename this to `SchemaLoader`? since what it does is just `Load`

Reply:
Or simply `Loader` as Go. I read that the expected name of the interface with just a method is `methoer` and outside the package, the interface, would be `schema.Loader` anyway

Reply:
Yes, `Loader` is nice. Go also likes short and sweet names.